css/roboto-fontface.scss

```diff
@@ -28,14 +29,18 @@ $roboto-font-path: '../fonts' !default;
     }
 }
 
-@mixin roboto-font-pair($type, $weight) {
+@mixin roboto-font-pair($type, $weight, $with-italic: true) {
     @include roboto-font($type, $weight);
-    @include roboto-font(#{$type}Italic, $weight, italic);
+    @if $with-italic {
+        @include roboto-font(#{$type}Italic, $weight, italic);
+    }
 }
 
-@include roboto-font-pair('Thin', 100);
-@include roboto-font-pair('Light', 300);
-@include roboto-font-pair('Regular', 400);
-@include roboto-font-pair('Medium', 500);
-@include roboto-font-pair('Bold', 700);
-@include roboto-font-pair('Black', 900);
+@if $include-styles {
```

Comment:
Ah no, the italic switch can also be dropped this way. When importing the `_mixins.scss` you can again choose to use either `@include roboto-font-pair` or just `@include roboto-font` if you do not need the italic version.